.github/workflows/tag.yml

```diff
@@ -0,0 +1,15 @@
+name: Tag version
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  call-bump-version-workflow:
+    uses: ASFHyP3/actions/.github/workflows/reusable-bump-version.yml@v0.6.0
+    with:
+      user: dbekaert
+      email: bekaertdavid@gmail.com
```

Comment:
@dbekaert for ACCESS19, we setup a "bot" user which creates the release tag, releases, and syncs `main` back to `dev` if a release branch was used. 

Here I've put you as this repo is under your personal GH account. That means it'll look like you did the tagging/releasing/syncing yourself. For this all to work, you'll need to create a personal access token with write access to this repo and add it as a secret named `RELEASE_ACCESS_TOKEN` to this repo.

Reply:
Ok i will do it in the morning.

Reply:
I would create a jpl org email e.g. `raider@jpl.nasa.gov` and have you, Jeremy, and Joe be the admins on this mailing list. This can then function as your bot - only need to create a github account for this email.